catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -19,33 +19,31 @@
             name="Result Count"
             description="Specifies the number of results to request from each source"
             type="Integer"
-            default="250"
-            required="true"/>
+            default="250"/>
```

Comment:
❓ Curious why some of these are no longer set to required. 
I imagine they were probably never truly required in the first place...

Reply:
Attribute descriptors are required by default, so there's no need to explicitly include `required="true"`.